src/armonik_cli/admin.py

```diff
@@ -43,37 +15,36 @@ def create_channel(arguments : dict):
     Returns:
         grpc.Channel: gRPC channel for communication
     """
-    if(arguments["--ca"] != None and arguments["--cert"] != None and arguments["--key"] != None):
-        ca = open(arguments["--ca"], 'rb').read()
-        cert = open(arguments["--cert"], 'rb').read()
-        key = open(arguments["--key"], 'rb').read()
+    if (arguments.ca != None and arguments.cert != None and arguments.key != None):
+        ca = open(arguments.ca, 'rb').read()
+        cert = open(arguments.cert, 'rb').read()
+        key = open(arguments.key, 'rb').read()
         credentials = grpc.ssl_channel_credentials(ca, key, cert)
-        return grpc.secure_channel(arguments["--endpoint"], credentials)
+        return grpc.secure_channel(arguments.endpoint, credentials)
     else:
-        return grpc.insecure_channel(arguments["--endpoint"])
-
-
-def create_session_filter(all: bool =False, running: bool =False, cancelled: bool =False) -> Filter:
-    """
-    Create a session Filter
-
-    Args:
-        all (bool): Show all sessions
-        running (bool): Show only running sessions
-        cancelled (bool): Show only cancelled sessions
-    Returns:
-        Filter object
-    """
-    if all:
-        session_filter = (SessionFieldFilter.STATUS == SESSION_STATUS_RUNNING) | (SessionFieldFilter.STATUS == SESSION_STATUS_CANCELLED)
-    elif running:
-        session_filter = SessionFieldFilter.STATUS == SESSION_STATUS_RUNNING
-    elif cancelled:
-        session_filter = SessionFieldFilter.STATUS == SESSION_STATUS_CANCELLED
-    else:
-         raise ValueError("SELECT ARGUMENT [--all | --running | --cancelled]")
-
-    return session_filter
+        return grpc.insecure_channel(arguments.endpoint)
+
+
+# def create_session_filter(all: bool = False, running: bool = False, cancelled: bool = False) -> Filter:
```

Comment:
You should remove it if you don't need it anymore. Git is there for tacking this